deploy/examples/import-external-cluster.sh

```diff
@@ -152,6 +171,54 @@ function importRGWAdminOpsUser() {
   --from-literal=secretKey="$RGW_ADMIN_OPS_USER_SECRET_KEY"
 }
 
+function createRBDStorageClass() {
+cat << eof | kubectl create -f -
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: $RBD_STORAGE_CLASS_NAME
+provisioner: rook-ceph.rbd.csi.ceph.com
+parameters:
+  clusterID: $NAMESPACE
+  pool: $RBD_POOL_NAME
+  imageFormat: "2"
+  imageFeatures: layering
+  csi.storage.k8s.io/provisioner-secret-name: $CSI_RBD_PROVISIONER_SECRET_NAME
+  csi.storage.k8s.io/provisioner-secret-namespace: $NAMESPACE # namespace:cluster
+  csi.storage.k8s.io/controller-expand-secret-name:  $CSI_RBD_PROVISIONER_SECRET_NAME
+  csi.storage.k8s.io/controller-expand-secret-namespace: $NAMESPACE # namespace:cluster
+  csi.storage.k8s.io/node-stage-secret-name: $CSI_RBD_NODE_SECRET_NAME
+  csi.storage.k8s.io/node-stage-secret-namespace: $NAMESPACE # namespace:cluster
+  csi.storage.k8s.io/fstype: ext4
+allowVolumeExpansion: true
+reclaimPolicy: Delete
+eof
+}
+
+function createCephFSStorageClass() {
+cat << eof | kubectl create -f -
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: $CEPHFS_STORAGE_CLASS_NAME
+provisioner: rook-ceph.cephfs.csi.ceph.com
```

Comment:
provisioner: $OPERATOR-NAMESPACE.cephfs.csi.ceph.com. the provisioner always starts with the namespace where the Rook operator is running.

Reply:
okay